modules/diskq/tests/test_diskq_truncate.c

```diff
@@ -177,22 +177,21 @@ test_diskq_truncate_without_diskbuffer_used(void)
   });
 }
 
-
-int
-main(void)
+static void
+setup(void)
 {
   msg_init(TRUE); // internal messages will go to stderr(), msg_init() is idempotent
   app_startup();
 
   configuration = cfg_new_snippet();
-  assert_true(cfg_init(configuration), "cfg_init failed!");
-
-  // Diskbuffer is the part of disk-queue that is used only when qout is full
-  test_diskq_truncate_with_diskbuffer_used();
-  test_diskq_truncate_without_diskbuffer_used();
+  cr_assert(cfg_init(configuration), "cfg_init failed!");
+}
 
+static void
+teardown(void)
+{
   app_shutdown();
   cfg_free(configuration);
-
-  return 0;
 }
+
+TestSuite(diskq_truncate, .init = setup, .fini = teardown);
```

Comment:
This test case fails in travis please fix it.

Reply:
ok..